cmd/sbom.go

```diff
@@ -0,0 +1,116 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/kondukto-io/kdt/client"
+	"github.com/kondukto-io/kdt/klog"
+	"github.com/spf13/cobra"
+)
+
+// sbomCmd represents the sbom root command
+var sbomCmd = &cobra.Command{
+	Use:   "sbom",
+	Short: "base command for SBOM(Software bill of materials) imports",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			_ = cmd.Help()
+			qwm(ExitCodeSuccess, "")
+		}
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(sbomCmd)
+
+	sbomCmd.AddCommand(importSbomCmd)
+
+	importSbomCmd.Flags().StringP("file", "f", "", "SBOM file to be imported. currently only .json format is supported")
+	importSbomCmd.Flags().StringP("project", "p", "", "kondukto project id or name")
+	importSbomCmd.Flags().StringP("repo-id", "r", "", "URL or ID of ALM repository")
+	importSbomCmd.Flags().StringP("sbom-type", "s", "", "Custom type(optional). Passing a different value than existing type(i.e application, container etc.) is advised")
+	importSbomCmd.Flags().StringP("branch", "b", "", "branch name for the project receiving the sbom")
+}
+
+// importSbomCmd represents the sbom import command
+var importSbomCmd = &cobra.Command{
+	Use:   "import",
+	Short: "imports sbom file to Kondukto",
+	Run:   importSbomRootCommand,
+}
+
+func importSbomRootCommand(cmd *cobra.Command, _ []string) {
+	// Initialize Kondukto client
+	c, err := client.New()
+	if err != nil {
+		qwe(ExitCodeError, err, "could not initialize Kondukto client")
+	}
+	sbomImport := SBOMImport{
+		cmd:    cmd,
+		client: c,
+	}
+	err = sbomImport.sbomImport()
+	if err != nil {
+		qwe(ExitCodeError, err, "failed to import sbom file")
+	}
```

Comment:
You should to use inline if
`if err := sbomImport.sbomImport(); err != nil`

Reply:
Fixed